appdatabase/database.go

```diff
@@ -26,34 +26,58 @@ var customSteps = []sqlite.PostStep{
 	{Version: 1687193315, CustomMigration: migrateWalletTransferFromToAddresses, RollBackVersion: 1686825075},
 }
 
-// InitializeDB creates db file at a given path and applies migrations.
-func InitializeDB(path, password string, kdfIterationsNumber int) (*sql.DB, error) {
-	db, err := sqlite.OpenDB(path, password, kdfIterationsNumber)
+// initializeUnencryptedDB creates unencrypted db file at a given path and applies migrations.
```

Comment:
hmm where is this used? I thought we determined having any unencrypted data could lead to privacy issues.